### PR TITLE
Allow ActiveAdmin 3

### PR DESCRIPTION
**Pull request by alejandroperea**
**ActiveAdmin 3.0 has been released** but the update is blocked for me due this gem does not support it.

Let's relax the condition to support ActiveAdmin 3.

### CODE REVIEW

**Comment by alejandroperea** [comment]
@blocknotes Could you take a look into this?

Thanks!

File: activeadmin_quill_editor.gemspec [hunk 1]
```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.files         = Dir['{app,lib}/**/*', 'LICENSE.txt', 'Rakefile', 'README.md']
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'activeadmin', '~> 2.0'
+  spec.add_runtime_dependency 'activeadmin', '>= 2.0, < 4'
```
Comment (blocknotes) [inline activeadmin_quill_editor.gemspec:28]: Hey @alejandroperea!
Thank you for your PR.

Can you update the dependency versions syntax?

It should be:

```suggestion
  spec.add_runtime_dependency 'activeadmin', '>= 2.0', '< 4'
```

**Review by blocknotes — CHANGES_REQUESTED** [review]

File: activeadmin_quill_editor.gemspec (hunk 1 above)
Comment (alejandroperea) [reply activeadmin_quill_editor.gemspec]: Whoah, good catch 😓 

I've also forgot to update the appraisals. I did another commit for updating them.

Comment (blocknotes) [reply activeadmin_quill_editor.gemspec]: Yeah, I see your changes only now 😅 

I'm working to update the appraisal config too.
Some configs are outdated.

Comment (blocknotes) [reply activeadmin_quill_editor.gemspec]: But I'll address them in another PR, your changes looks good 👍

**Review by blocknotes — APPROVED** [review]
Thanks for the PR 💪

**Pull request merged**